site/public/css/server.css

```diff
@@ -763,6 +763,26 @@ td > a.active-sort {
     background-color: var(--standard-medium-blue);
 }
 
+.dark-green-background{
+    background-color: #00ff11;
```

Comment:
Use colors defined in colors.css to ensure that these changes match the pre-existing color scheme of submitty.

Reply:
Okay. I will change the colors to colors from colors.css.